Task1/script.js

```diff
@@ -0,0 +1,55 @@
+/*let randn_bm = function() {  
+    let u = 0, v = 0;  
+        while (u === 0) 
+            u = Math.random();  
+        while (v === 0) 
+            v = Math.random();  
+    return Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2 | 0;
+};*/
+
+const array = [];
+const result = {};
+
+let randn_bm = function() {  
+    let u = 0, v = 0;  
+        while (u === 0) 
```

Comment:
I think you need to pay more attention code format. Because all offsets should be equal if we start some new block of code. for example
```
let u = 0, v = 0;  
while (u === 0)
    u = Math.random();
```